src/tests/helpers.js

```diff
@@ -782,6 +782,25 @@ export const schemaInputWithStatement = {
   },
 };
 
+export const schemaHiddenInputWithStatement = {
```

Comment:
> @sandrina-p I've added the schema for the upcoming docs in https://github.com/remoteoss/json-schema-form/commit/623c31b5c5d2aedfb1e6ad7f070e3453909cb878

@danielcardoso5 I want your opinion here. Do you agree with this pattern of creating a schema but not used by the codebase directly? I find it very odd... I'd rather simply write in our internal codebase. If it gets stale/invalid the tests will fail anyway.

Context: When we open-sourced JSF we didn't spend much time in this file because of speed/velocity. And now we are paying the bill (engineers following a pattern that doesn't make sense.)

Reply:
To clarify, I don't expect you to fix/find-the-perfect-solution right now. Just want your opinion, and then we can tackle in later on.

Reply:
@sandrina-p yeah it's odd, this is just a symptom of the documentation leaving separately from this repo, if we moved it over this wouldn't be a problem, but you already know this 😄 

We could move the ones used for the documentation to our internal codebase, one immediate advantage is that you don't need to do anything in this repo if you just want to improve the docs by adding or changing an example. In this case I added the schema right away to avoid creating another PR just for it.